cstor-operator.yaml

```diff
@@ -336,7 +467,7 @@ metadata:
   labels:
     name: cspc-operator
     openebs.io/component-name: cspc-operator
-    openebs.io/version: 1.11.0
+    openebs.io/version: 1.12.0
```

Comment:
Why do we need to change the version for this file?

Reply:
These are the files which are maintained outside of  version dir,  similar to openebs-operator or other yaml, as a latest version release yaml